PyRDF/backend/Utils.py

```diff
@@ -1,10 +1,41 @@
-import ROOT
-import os
 import logging
+import multiprocessing
+import os
+import threading
+
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+
+import ROOT
 
 logger = logging.getLogger(__name__)
 
 
+def run_graphs(proxies, numthreads=multiprocessing.cpu_count()):
+    """
+    Trigger multiple RDF graphs through multithreading.
+    """
+    ROOT.EnableThreadSafety()
```

Comment:
Why `EnableThreadSafety` and not `EnableImplicitMT` here?

Reply:
There is no implict multithreading here, I am explicitly creating the threads and giving each of them a function to run. `EnableThreadSafety` is there just for *safety* (stuff like avoiding the modification of `gDirectory` from different threads that trigger jobs that save data to different ROOT files), I'll try to remove it and see if it makes a difference.

Reply:
Right, there would be IMT if the selected backend was the local one. I think my comment of implementing this function per-backend goes in the right direction.